src/style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.page {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  justify-content: center;
+
+  width: 200px;
+  min-height: 406px;
```

Comment:
try to ged rid of min-height
content will take as much space as it needs
for now your elemen looks way to long in test

Reply:
@MaxFedak  does not work